libraries/pfft/README

```diff
@@ -0,0 +1,15 @@
+PFFT (Parallel Fast Fourier Transform) is a software library for
```

Comment:
Same here willy since it needs fftw-mpi. Not sure if this is a good idea to this package willy. I will delegate to your opinion here.